docs/spec.md

```diff
@@ -1988,7 +2001,8 @@ If `feature_group_count = 1` and `batch_group_count = 1`, then for all
 `output_spatial_index` in the index space of `dim(result, output_spatial_dimensions)`,
 `result[result_shape(:, output_spatial_index, :)] = dot_product` where:
 
-* `padded_lhs = pad(lhs, 0, lhs_padding[:, 0], lhs_padding[:, 1], lhs_base_dilations[:] - 1)`.
+* `padded_lhs_value = is_quantized_tensor(lhs)? zero_point(lhs) : 0`.
```

Comment:
A minor comment: add a space before the question mark:

* `padded_lhs_value = is_quantized_tensor(lhs) ? zero_point(lhs) : 0`.